ext/mkrf_conf.rb

```diff
@@ -0,0 +1,24 @@
+require 'rubygems'
+require 'rubygems/command.rb'
+require 'rubygems/dependency_installer.rb' 
+begin
+  Gem::Command.build_args = ARGV
+  rescue NoMethodError
+end 
+inst = Gem::DependencyInstaller.new
+
+
+begin
+  if RUBY_VERSION < "1.9.3"
+    inst.install "activerecord", ">2.0", "< 4.0"
+    inst.install 'i18n', "< 0.7"
+  else
+    inst.install "activerecord", "> 4.0"
```

Comment:
I'm not sure this is true.

We want to ensure that if activerecord >= 4.x then the ruby interpreter is >= 1.9.3